specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/virtualMachineExamples/VirtualMachine_Get_InstanceView.json

```diff
@@ -0,0 +1,139 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-09-01",
+    "vmName": "myVM"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
> "body": { [](http://example.com/codeflow?start=5&length=10)

should this also include the etag version of the VM that the instance was created from?

Reply:
hello. it could have it, but it also doesn't have to. its getting added to 4 other examples so I think its okay to not have it in some places https://github.com/Azure/azure-rest-api-specs/pull/26368/files